docs/tutorials/notes-mvvm/index.yml

```diff
@@ -0,0 +1,56 @@
+### YamlMime:Tutorial
+title: "Upgrade your app with MVVM concepts"
+metadata:
+  title: "Tutorial: Use the MVVM toolkit to update the Notes app"
+  description: Upgrade the Notes app from the previous tutorial with the MVVM Community Toolkit.
+  audience: Developer
+  level: Beginner
+  ms.topic: interactive-tutorial
+  ms.date: 12/21/2022
+  author: adegeo
+  ms.author: adegeo
+items:
+- durationInMinutes: 1
+  content: |
+    [!INCLUDE [](includes/intro.md)]
+
+- title: Upgrade the app from .NET 6 to .NET 7
+  durationInMinutes: 3
+  content: |
+    [!INCLUDE [](includes/upgrade.md)]
+
+- title: Update the model
+  durationInMinutes: 7
+  content: |
+    [!INCLUDE [](includes/model.md)]
+
+- title: About viewmodel and About view
+  durationInMinutes: 10
+  content: |
+    [!INCLUDE [](includes/viewmodel-about.md)]
+
+- title: Note viewmodel and Note view
+  durationInMinutes: 15
+  content: |
+    [!INCLUDE [](includes/viewmodel-note.md)]
+
+- title: Notes viewmodel and AllNotes view
+  durationInMinutes: 10
+  content: |
+    [!INCLUDE [](includes/viewmodel-notes.md)]
+
+- title: Fix the final bugs
+  durationInMinutes: 5
+  content: |
+    [!INCLUDE [](includes/bugs.md)]
+
+- content: |
+    Your app is now using MVVM patterns!
```

Comment:
Does the acronym need expanding?

Reply:
I don't think so, by the time they reach this position, they should know what MVVM is.